### ISSUE TITLE
Kind load error

### ISSUE THREAD

**Issue by luisfcofv** [spring-guides/top-spring-on-kubernetes#3]
I'm following this [guide](https://spring.io/guides/topicals/spring-on-kubernetes/), and I'm hitting this error when loading the docker image.

```sh
kind load docker-image spring-k8s/gs-spring-boot-k8s:snapshot
Image: "spring-k8s/gs-spring-boot-k8s:snapshot" with ID "sha256:cb9e9a6289f812a147e95d8447fe65526e1f256154901d37c5c5e24409704130" not yet present on node "kind-control-plane", loading...
ERROR: failed to load image: command "docker exec --privileged -i kind-control-plane ctr --namespace=k8s.io images import --digests --snapshotter=overlayfs -" failed with error: exit status 1
```

I'm using an Apple M1 machine, so this might be related.
Spring boot version: 2.7.1

Any ideas on how to solve it?

Thank you

**Comment by ryanjbaxter** [comment]
Seems like it might be related to using an M1 and Kind.  There are some workaround here 
https://github.com/kubernetes-sigs/kind/issues/2402

**Comment by ryanjbaxter** [comment]
Fixed via #7
